cli/internal/cmd/configfetchmeasurements.go

```diff
@@ -79,6 +88,46 @@ func configFetchMeasurements(cmd *cobra.Command, fileHandler file.Handler, clien
 	return nil
 }
 
+func verifyWithRekor(cmd *cobra.Command, hash string) error {
+	r, err := sigstore.NewRekor()
```

Comment:
```suggestion
	rekor, err := sigstore.NewRekor()
```

https://github.com/edgelesssys/wiki/blob/master/company/gostyle.md#variable-names